radiotools/atmosphere/models.py

```diff
@@ -834,7 +839,7 @@ def _get_vertical_height(self, zenith, X, observation_level=0):
         tmp = np.zeros_like(zenith)
 
         if np.sum(mask_numeric):
-            # print("get vertical height numeric", zenith)
+            # logger.info("get vertical height numeric", zenith)
```

Comment:
I would either remove theses commented statements or make them `debug` (if you do that please use `"bla {}".format()` instead of f-strings